File: RedSheet.DbEntities/Models/vUserMessage.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vUserMessages", Schema = "dbo")]
	[Serializable]
	public partial class vUserMessage  
	{
 
	[Required] 
	[MaxLength(100)]
	public string CreatedBy { get; set; } 

	public Nullable<System.DateTime> CreatedOn { get; set; } 
 
	[Required]
	public string Message { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string Status { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string UpdatedBy { get; set; } 

	public Nullable<System.DateTime> UpdatedOn { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int UserMessageId { get; set; } 

	public vUserMessage()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-user-message.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vUserMessage {
    constructor(vUserMessage?: vUserMessage )  {
        let properties = [ "createdBy", "createdOn", "message", "status", "updatedBy", "updatedOn", "userMessageId",];
        for (let property of properties)
            if (vUserMessage && vUserMessage[property])
                this[property] = vUserMessage[property];
    }
 
    @required()
    @maxLength(100)
	createdBy : string =   undefined;
 
	createdOn : Date =   undefined;
 
    @required()
	message : string =   undefined;
 
    @required()
    @maxLength(100)
	status : string =   undefined;
 
    @required()
    @maxLength(100)
	updatedBy : string =   undefined;
 
	updatedOn : Date =   undefined;
 
	userMessageId : number =   0 ;


}
